initdb.py

```diff
@@ -165,10 +161,15 @@ def main(username='', password=''):
 
     # Close cursor
     mysql_cursor.close()
+    
     # Close connection
     mysql_connection.close()
 
     # Print completed
     print("Database set up completed.")
 
-if __name__ == '__main__': main()
+if __name__ == '__main__':
+    try:
+        main()
+    except Exception as InitDbException:
```

Comment:
`InitDbException` violates [Google Python Style Guide: Naming](https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming)

make it something like: `db_exception`

I also would recommend placing a TODO here to mark that in the future exception handling here shouldn't be so broad.